src/drivers/imx/edma.c

```diff
@@ -195,10 +198,14 @@ static int edma_stop(struct dma_chan_data *channel)
 
 static int edma_copy(struct dma_chan_data *channel, int bytes, uint32_t flags)
 {
-	struct dma_cb_data next = { .elem.size = bytes };
+	struct dma_cb_data next = {
+		.channel = channel,
+		.elem.size = bytes
```

Comment:
not a reason for a new version on its own, but if you do have to make a new version, maybe you can also add a comma above after `bytes` (and in similar places, that your patches touch, if any). As we have seen recently it makes potential future patching easier.

Reply:
done